packages/databyss-data/pouchdb/utils.ts

```diff
@@ -11,6 +12,28 @@ export const addTimeStamp = (doc: any): any => {
   return { ...doc, createdAt: Date.now() }
 }
 
+// const EM = new EventEmitter()
+
+// EM.on('foo', () => {
+//   console.log('some code')
+// })
+
+// EM.emit('foo')
+
+interface Patch {
+  $type: string
+  _id: string
+  doc: any
+}
+
+type upsertDictionary = {
+  current: Patch[]
+}
+
+export const upQdict: upsertDictionary = {
```

Comment:
this isn't a dictionary, and the module const should be named like a ref, like we discussed, so
`export const upsertQueueRef: { current: Patch[] } = {`